readme.md

```diff
@@ -68,495 +35,38 @@ From the npm registry:
 > yarn/pnpm add express-rate-limit
 ```
 
-From Github Releases:
-
-```sh
-# Using npm
-> npm install https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-# Using yarn or pnpm
-> yarn/pnpm add https://github.com/express-rate-limit/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
-```
-
-Replace `{version}` with the version of the package that you want to your, e.g.:
-`6.0.0`.
-
-## Usage
-
-### Importing
-
-This library is provided in ESM as well as CJS forms, and works with both
-Javascript and Typescript projects.
-
-**This package requires you to use Node 16 or above.**
-
-Import it in a CommonJS project (`type: commonjs` or no `type` field in
-`package.json`) as follows:
-
-```ts
-const { rateLimit } = require('express-rate-limit')
-```
-
-Import it in a ESM project (`type: module` in `package.json`) as follows:
-
-```ts
-import { rateLimit } from 'express-rate-limit'
-```
-
-### Examples
+> Note that this library requires you to use `node` version 16 or above.
```

Comment:
I think we can remove the installation instructions and min node version from the readme and move them to the full docs.